apps/website/src/routes/sudoku/editor/+page.svelte

```diff
@@ -75,21 +81,24 @@
     let solution: RouterInputs['sudokus']['provideSolutionToPuzzle']['solution'] = undefined;
     // create solution
     if (provideSolution) {
-      let values = userInputs.values;
+      let cellValues = gameData.cellValues;
       if (walkthrough.length) {
-        const finalStep = walkthrough[walkthrough.length - 1].step;
+        const finalStep = walkthrough[walkthrough.length - 1].gameData;
         if (
-          userInputs.values.some((row, i) => {
-            return row.some((value, j) => {
-              return value === '' && finalStep.values[i][j] !== '';
+          gameData.cellValues.some((row, i) => {
+            return row.some((cell, j) => {
+              return !cell.digits && finalStep.cellValues[i][j].digits;
```

Comment:
```suggestion
              return cell.digits !== "" && finalStep.cellValues[i][j].digits;
```
Or what is meant here